src/cli/commands/pubsub/ls.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+
+const utils = require('../../utils')
+const debug = require('debug')
+const log = debug('cli:pubsub')
+log.error = debug('cli:pubsub:error')
+
+module.exports = {
+  command: 'ls',
+
+  describe: 'Get your list of subscriptions',
+
+  builder: {},
+
+  handler (argv) {
+    utils.getIPFS((err, ipfs) => {
+      if (err) {
+        throw err
+      }
+
+      ipfs.pubsub.ls((err, subscriptions) => {
+        if (err) {
+          throw err
+        }
+
+        console.log(JSON.stringify(subscriptions, null, 2))
```

Comment:
`JSON.stringify` does not create a very nice user output, this should rather iterate through the subs and print them in a nice format.